pkg/dashboard/worker/gitops.go

```diff
@@ -924,54 +945,32 @@ func saveAndBroadcastGitopsCommit(
 	}
 }
 
-func gitopsRepoForEnv(db *store.Store, env string) (string, bool, error) {
+func gitopsRepoForEnv(db *store.Store, env string) (string, bool, bool, error) {
```

Comment:
Soon we return the whole environment.

Time to refactor this function when it has 4 return values.

Why not just return `*model.Environment`.
Maybe it's not the right solution, but some struct is, with a different function name.